packages/components/src/components/dataDisplay/Badge/Badge.tsx

```diff
@@ -1,5 +1,28 @@
 import React, { FC } from 'react';
-import { Badge as ChakraBadge } from '@chakra-ui/react';
+import { Badge as ChakraBadge, IconProps } from '@chakra-ui/react';
+import { Flex } from '@components';
 import { BadgeProps } from './types';
 
-export const Badge: FC<BadgeProps> = (props) => <ChakraBadge {...props} />;
+export const Badge: FC<BadgeProps> = ({ leadingIcon, trailingIcon, children, ...props }) => {
+  const leftIcon = getIcon(leadingIcon);
+  const rightIcon = getIcon(trailingIcon);
+  return (
+    <ChakraBadge {...props}>
+      <Flex alignItems="center" gap="spacer-1">
```

Comment:
Could you try moving these styles to the `ChakraBadge`?

Reply:
Fixed!

Reply:
Is the `inline-flex` needed? Or would it work with display flex?